### PR TITLE
Add tools to connect to HSQLDB CLI

### PR DESCRIPTION
**Pull request by mboussaa**

### CODE REVIEW

File: tools/hsqldb-connection/sqltool.rc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+urlid scheduler
+
+url jdbc:hsqldb:hsql://localhost:9001/scheduler
+
+username scheduler
+
+password
```
Comment (fviale) [inline tools/hsqldb-connection/sqltool.rc:7]: there should be a new line

File: tools/hsqldb-connection/db-connect.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# A Simple Shell Command To access HSQLDB CLI and connect by default to scheduler DB
+# If you would like to connect to another ProActive DB, please configure the sqltool.rc file before running the script
+# If java is not set on your system, please use java under [Schedulering_Path]/jre/bin/java
+
+java -jar sqltool-2.4.1.jar --rcFile=sqltool.rc scheduler
```
Comment (fviale) [inline tools/hsqldb-connection/db-connect.sh:6]: a new line is missing

File: tools/hsqldb-connection/db-connect.sh [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```
Comment (fviale) [inline tools/hsqldb-connection/db-connect.sh:1]: you should also create a corresponding .bat script

Comment (mboussaa) [reply tools/hsqldb-connection/db-connect.sh:1]: ok done. Could you check the .bat ?

File: tools/hsqldb-connection/db-connect.sh (hunk 2 above)
Comment (fviale) [inline tools/hsqldb-connection/db-connect.sh:6]: actually for both windows and linux, you should use the jre present in ProActive

Comment (fviale) [reply tools/hsqldb-connection/db-connect.sh:6]: You can look at proactive-node.bat and proactive-node to see how the APP_HOME folder is determined.
Example in proactive-node.bat:
`set DIRNAME=%~dp0
if "%DIRNAME%" == "" set DIRNAME=.
set APP_BASE_NAME=%~n0
set APP_HOME=%DIRNAME%..`

**Pull request merged**